docs/UserGuide.md

```diff
@@ -380,18 +395,16 @@ Examples:
 
 ### Finding events by name : `/e find`
 
-Find events with names containing the given keyword.<br>
-If there are no volunteers whose names match the keyword in the database, the entire list of volunteers will be displayed.
+Looking for a particular event? This command helps you find it by its name.
 
 Format: `/e find KEYWORD`
 
-* The search is case-insensitive.
-* Only the name is searched.
-* The searching algorithm returns events with any part of their name matching the `KEYWORD`
-    * e.g. `Group` will match `Group Project`, `Grouping Task` and `Group`.
+* It doesn't matter if you use capital or small letters
+* It finds partial matches too - typing `food` can find `Food Drive` or `Seafood Party`
+* If nothing matches, you'll see all events again
```

Comment:
Would something like "but dont worry, you'll get an error message letting you know!" be gd here?

Reply:
yep, included it